invenio_rdm_records/records/systemfields/tombstone.py

```diff
@@ -148,6 +149,22 @@ def __repr__(self):
 class TombstoneField(SystemField):
     """System field for accessing a record's deletion status."""
```

Comment:
```suggestion
    """System field for accessing a record's deletion status.
    
    Note: To remove a record's tombstone, use ``del record.tombstone``.
    This will take care of properly de-initializing the information.
    """
```